tensorflow/lite/micro/cortex_m_corstone_300/README.md

```diff
@@ -0,0 +1,35 @@
+
+# Running a fixed virtual platform based on Corstone-300 software
+
+This target makes use of a fixed virtual platform (FVP) based on Arm Cortex-300 based software.
+More info about Arm Corstone-300 software: https://developer.arm.com/ip-products/subsystem/corstone/corstone-300
+More info about FVPs: https://developer.arm.com/tools-and-software/simulation-models/fixed-virtual-platforms
+
+To fullfill the needed requirements it is depending the following projects:
+* Arm Ethos-U Core Platform: https://review.mlplatform.org/admin/repos/ml/ethos-u/ethos-u-core-platform
```

Comment:
Please clarify what dependency we have on theses repos

Reply:
Ok